source/reference/config/triggers.txt

```diff
@@ -96,15 +118,26 @@ Database Triggers
 
 Database trigger configurations conform to the base trigger schema with
 additional configuration options that specify which collection to watch and when
-to fire the trigger. The following fields exist in *database* and *deployment* 
-trigger configuration files:
+to fire the trigger. 
+
+Below are two configurations: one configuration is 
+for a trigger with a Function event processor and the other is for a trigger with 
+an AWS EventBridge event processor. The only difference between the two configuraitons
```

Comment:
```suggestion
an AWS EventBridge event processor. The only difference between the two configurations
```

Reply:
Deleted this sentence because your above suggestion made it more succinct! This is extra detail they can see